src/modules/services_manager_target.rb

```diff
@@ -172,6 +176,16 @@ def changes_summary
 
   private
 
+    def ensure_target
+      return if Yast2::Systemd::Target.find(self.default_target)
+
+      # TRANSLATORS: error popup, %s is the default target e.g. graphical
+      Report.Warning(_("Target '%s' is not available." \
+                       " Using text mode as default target.") % default_target)
+      self.default_target = BaseTargets::MULTIUSER
```

Comment:
The self is needed because ruby is taking default_target as a local variable here. That's why I have set the rest to self.... too in order to be consequent. I have no idea why ruby is so here. Just try it out in the testcase :-)

Reply:
As commented in IRC, here it is needed because in the setter is not clear if you want to invoke the method or to define a new local variable (ruby stuff xD).